internal/proto/local/controller/api/services/v1/session_service.proto

```diff
@@ -63,6 +63,7 @@ message GetSessionResponse {
 message ListSessionsRequest {
 	string scope_id = 1;
 	bool recursive = 20 [json_name="recursive"];
+	string filter = 30 [json_name="filter"];
```

Comment:
Is there a reason you put `json_name` here and not in other places?

Reply:
Nope.  I've added the option in all the .proto files.